test/src/main/java/org/apache/accumulo/test/server/security/SystemCredentialsIT.java

```diff
@@ -85,8 +84,8 @@ public static void main(final String[] args) throws AccumuloException, TableNotF
           .as(creds.getPrincipal(), creds.getToken()).build()) {
         client.securityOperations().authenticateUser(creds.getPrincipal(), creds.getToken());
         try (Scanner scan = client.createScanner(RootTable.NAME, Authorizations.EMPTY)) {
-          for (Entry<Key,Value> e : scan) {
-            e.hashCode();
+          while (scan.iterator().hasNext()) {
+            scan.iterator().next();
           }
```

Comment:
This won't work... calling `iterator()` multiple times on the scanner creates a new scan each time. You'd have to save the iterator to a variable and keep calling hasNext and next on that variable. Alternatively, you can do almost anything else with the `e` variable from the original code. The only reason `e.hashCode()` was called was to avoid a warning about the unused variable, `e`. Now, you could probably do something like:

```suggestion
          scan.forEach((k,v) -> {});
```

If that generates a warning, you could maybe assign it to a variable: `private static final BiConsumer<Entry<Key,Value>> NOOP = (k,v) -> {};`

This same strategy could be used to replace other uses of Iterators.size, where we don't care about the size. That was only ever used as an easy way to iterate fully over the scanner without triggering a warning about unused variables. But, if it's causing warnings, it might be best to find a different solution that doesn't, instead of trying to chase the warnings by using the return value that we never cared about to begin with.